README.md

```diff
@@ -192,6 +192,6 @@ earthaccess is licensed under the MIT license. See [LICENSE](LICENSE.txt).
 
 <div><img src="https://raw.githubusercontent.com/nsidc/earthdata/main/docs/nsidc-logo.png" width="84px" align="left" text-align="middle"/>
 <br>
- This repository is not actively supported by NSIDC but we welcome issue submissions and pull requests in order to foster community contribution.
+ This repository is supported by a joint effort of NSIDC, NASA DAACs, and the Earth science community, and we welcome any contribution in the form of issue submissions, pull requests, or discussions. Issues labeled as https://github.com/nsidc/earthaccess/labels/good%20first%20issue are a great place to get started.
```

Comment:
```suggestion
 This repository is supported by a joint effort of NSIDC, NASA DAACs, and the Earth science community, and we welcome any contribution in the form of issue submissions, pull requests, or discussions. Issues labeled as ["good first issue"](https://github.com/nsidc/earthaccess/labels/good%20first%20issue) are a great place to get started!
```
I'd recommend markdown formatting the link in this sentence and adding an exclamation point because contributions are exciting! 

I also think linking to the project board, or roadmap is a good idea.

Reply:
Now I'm curious! I found recently that if you directly use a bare label URL like that one in a comment, it renders out as a beautiful link: https://github.com/nsidc/earthaccess/labels/good%20first%20issue 

Is that magic also works in GitHub Flavored Markdown _files_?

Reply:
Ooo, looking at the preview mode, the magic _does_ work! :magic_wand:

![image](https://github.com/nsidc/earthaccess/assets/3608264/2598a0d8-b69c-4991-935e-9a33040b86f5)

Reply:
OOH Nice! It totally makes sense it does that since issues/prs also do that magic, but I'd never seen it before. Disregard; bare link is better.

Reply:
Heh I originally did the same thing but it wasn't until I saw the Preview that I noticed the magic!